README.md

```diff
@@ -79,6 +79,47 @@ Note there are various caveats to using this, which you can read about in detail
 - https://github.com/pavouk-0/postgres-xl-docker/issues/27
 - https://github.com/pavouk-0/postgres-xl-docker/pull/28
 
+## Clustering (Kubernetes; Automatically)
+
+So as ```docker-stack``` doesn't support ```depends_on``` option, ```docker-compose.stack.yml``` uses ```wait-for-it``` script. It delay main command until GTM node is ready.
+
+Also ```pgpool``` was added to ```docker-compose.stack.yml```. It sees to **coordinators nodes** and is making base enter point.
+
+```sh
+# TO UP
+docker stack deploy --orchestrator=kubernetes --namespace=default -c docker-compose.stack.yml pxl_stack
+# After you deployed pxl_stack, you need to initialize it (if it didn't do early or you purged PVC)
+./bin/init-eg-stack
+
+# TO DOWN
+docker stack rm --orchestrator=kubernetes pxl_stack
+# Keep in mind that Persistent volumes (PVC) are NOT Docker volumes
+# If you want to purge that, you can to remove !!! ALL !!! PVC 👇
+kubectl delete pvc --all
+# To show all volumes use 👇
+kubectl get pv
+
+# TO PORT FORWARD
+kubectl port-forward pg-pool-5b64b65f48-mc7g5 5432:5432
```

Comment:
I'm not currently working with K8S so I can't check this, but this seems like interesting information. Thanks for being so detailed.

Reply:
To be honest I forgot to anonymize that string.